src/file-events/headers/apple_fsnotifier.h

```diff
@@ -32,10 +32,13 @@ class WatchPoint {
 
 class Server : AbstractServer {
 public:
-    Server(JNIEnv* env, jobject watcherCallback);
+    Server(JNIEnv* env, jobject watcherCallback, long latencyInMillis);
     ~Server();
 
-    void startWatching(const u16string& path, long latencyInMillis);
+    void startWatching(const u16string& path);
+    void stopWatching(const u16string& path);
+
+    // TODO This should be private
```

Comment:
Can we make it private? And maybe use `friend` for the static method which calls it?

Reply:
We will, yeah.